superset-frontend/src/features/home/Menu.tsx

```diff
@@ -186,6 +186,31 @@ export function Menu({
     return () => window.removeEventListener('resize', windowResize);
   }, []);
 
+  enum paths {
+    EXPLORE = '/explore',
+    DASHBOARD = '/dashboard',
+    CHART = '/chart',
+    DATASETS = '/tablemodelview',
+  }
+  const [activeTabs, setActiveTabs] = useState(['']);
```

Comment:
Is there a benefit to defaulting to an array with one empty string rather than an empty array?